models/truckModel.js

```diff
@@ -2,14 +2,18 @@
 // when exporting the mongoose model you will create using this schema, 
 // name your model 'Truck' (see the example in the slide deck for 'Book')
 // this is to match the name of the collection the data was imported into
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema();
```

Comment:
This should be mongoose.Schema; Schema is a constructor function, and you want to assign it to a shorter variable, Schema.